jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -199,6 +197,40 @@ protected ScopedContext newContext()
         return new ScopedContext();
     }
 
+    /**
+     * @return true if {@link #shutdown()} will wait for all requests to complete.
+     */
+    public boolean isGraceful()
+    {
+        return _graceful;
+    }
+
+    /**
+     * Set if this context should gracefully shutdown by waiting for all current requests to
+     * complete.  This can be set via the "graceful" property of the {@code DeploymentManager}.
```

Comment:
`DeploymentManager` has not properties that can be set, so the javadoc should be improved to say that this can be set at deployment time via a properties file with the same name of the webapp to deploy.

However, why we need this? `ContextHandler` implements `Graceful` so why we need this at all?

Furthermore, seems confusing to me to set `graceful=false`, but then be able to call `shutdown()` which is supposed to be graceful but it is not.